File: csharp/PrvniObjekt/PrvniObjekt/Clovek.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PrvniObjekt
{
    internal class Clovek
    {

        //atributy (promenne nalezici tride)
        public string jmeno;
        private int vek;

        //metody (funkce nalezici tride)
        public void Pozdrav()
        {
            Console.WriteLine($"Ahoj posila {jmeno}");
        }

        //pretizeni metody
        public void Pozdrav(string pozdrav)
        {
            Console.WriteLine($"{pozdrav} posila {jmeno}");
        }

        public string VratPozdrav(string pozdrav)
        {
            //Console.WriteLine($"Vraceny pozdrav: {pozdrav} posila {jmeno}");
            return $"Vraceny pozdrav: {pozdrav} posila {jmeno}";
        }
    }
}

File: java/PrvniObjekt/src/main/java/org/example/Clovek.java
package org.example;

public class Clovek {

    //atributy (promenne nalezici tride)
    public String jmeno;
    private int vek;

    //metody (funkce nalezici tride)
    public void pozdrav()
    {
        System.out.println("Ahoj posila " + jmeno);
    }

    //pretizeni metody
    public void pozdrav(String pozdrav)
    {
        System.out.println(pozdrav + " posila " + jmeno);
    }

    public String vratPozdrav(String pozdrav)
    {
        //Console.WriteLine($"Vraceny pozdrav: {pozdrav} posila {jmeno}");
        return "Vraceny pozdrav: " + pozdrav + " posila " + jmeno;
    }

}
